pilz_control/test/unittest_pilz_joint_trajectory_controller.cpp

```diff
@@ -348,34 +349,72 @@ testing::AssertionResult PilzJointTrajectoryControllerIsExecutingTest::invokeIsE
   return invoke_is_executing(controller_, result);
 }
 
-testing::AssertionResult PilzJointTrajectoryControllerIsExecutingTest::waitForIsExecutingResult(bool expectation,
-                                                                                                bool& result)
+testing::AssertionResult PilzJointTrajectoryControllerIsExecutingTest::waitForIsExecutingResult(const bool is_executing_expectation)
 {
   auto invoke_is_executing{ GetParam() };
-  ros::Duration update_period{ DEFAULT_UPDATE_PERIOD_SEC };
+  const ros::Duration update_period{ DEFAULT_UPDATE_PERIOD_SEC };
 
   while (ros::ok())
   {
     progressInTime(update_period);
     updateController();
 
-    auto assertion_result{ invoke_is_executing(controller_, result) };
+    bool is_executing_result {!is_executing_expectation};
+    auto assertion_result{ invoke_is_executing(controller_, is_executing_result) };
```

Comment:
The functions `invokeIsExecuting` and `invoke_is_executing` can be easily confused.

Reply:
Done.